Products/ZenUI3/browser/resources/js/zenoss/process.js

```diff
@@ -1132,25 +1132,36 @@ Ext.define("Zenoss.TestRegexDialog", {
                                     autoWidth: true
                                 }
                             ]},
-                            {layout: {
-                                type: 'hbox',
-                                pack: 'start',
-                                align: 'stretch'
-                            },
-                            items: [
-                                    {fieldLabel: 'File', html: '<input type="file" id="files" name="file" />', ui: 'dialog-dark', flex:1},
+                            {
+                                layout: {
+                                    type: 'hbox',
+                                    pack: 'start',
+                                    align: 'stretch'
+                                },
+                                items: [
+                                    {
+                                        fieldLabel: 'File',
+                                        xtype: 'filefield',
```

Comment:
Since Ext.JS has component `Ext.form.field.File` it is better to use it than create some custom file input using `html` property as it was before.